bundles/org.openhab.binding.myuplink/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="myuplink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="account">
+		<label>myUplink Account</label>
+		<description>Cloud connection to a myUplink account.</description>
+		<config-description-ref uri="thing-type:myuplink:account"/>
+	</bridge-type>
+	<thing-type id="genericDevice">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="account"/>
+		</supported-bridge-type-refs>
+		<label>myUplink Generic Device</label>
+		<description>Cloud connection to a myUplink device.</description>
+		<channels>
+			<channel id="command" typeId="rwtype-command">
+				<label>Generic Command</label>
```

Comment:
What is the purpose of this channel as commands can already be send to all channels using the native API? It looks like an alternative way of sending commands.

Reply:
There are many more channels existing than exposed by the API. Although there is no way to get their current value it is still possible to send data to them and update settings. Of course this is mainly for advanced users.

Reply:
Still dont’t understand, when do you use this channel? And why would you not model it into a regular channel.

Reply:
All channels a created dynamically from the API so there is no hard-coded modelling of channels.
In addition to those channels which are officially exposed by the API there are some "hidden channels". You cannot read them so the API will not expose any values but the API accepts setting values. As I already wrote in our discussion regarding documentation of channels in the readme this API is very generic and supports a huge number of heat pump from different vendors. This simple implementation allows advanced users who know those hidden channels to change theirs values. I did not find a better solution to implement such a generic way to support arbitrary hidden channels.

Reply:
In that case the channel might be marked as advanced.

Reply:
the channel-type is already marked as advanced and thus the channel itself is also not shown by default.